includes/admin/payments/view-order-details.php

```diff
@@ -18,6 +18,11 @@
  * @since 1.6
  * @since 3.0 Updated to use the new EDD\Orders\Order object.
  */
+
+wp_enqueue_script( 'edd-admin-orders' );
+// Enqueued for backwards compatibility. Empty file.
+wp_enqueue_script( 'edd-admin-payments' );
+
```

Comment:
This can be moved under the potential `wp_die()` calls so they don't get enqueued until we know we have something to display.